components/brave_perf_predictor/browser/perf_predictor_tab_helper.h

```diff
@@ -0,0 +1,76 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_PERF_PREDICTOR_BROWSER_PERF_PREDICTOR_TAB_HELPER_H_
+#define BRAVE_COMPONENTS_BRAVE_PERF_PREDICTOR_BROWSER_PERF_PREDICTOR_TAB_HELPER_H_
+
+#include <memory>
+#include <string>
+
+#include "base/macros.h"
+#include "brave/components/brave_perf_predictor/browser/bandwidth_savings_predictor.h"
+#include "brave/components/p3a/buildflags.h"
+#include "components/page_load_metrics/common/page_load_metrics.mojom.h"
+#include "components/prefs/pref_registry_simple.h"
+#include "content/public/browser/navigation_handle.h"
```

Comment:
includes like this should be replaced with forward declarations. You don't need to include headers for incomplete types (like pointer or reference to something)